draft-ietf-quic-transport.md

```diff
@@ -1462,11 +1462,14 @@ The draining period persists for three times the current Retransmission Timeout
 can be acknowledged, but no new application data can be sent on the connection.
 
 Different treatment is given to packets that are received while a connection is
-in the draining period depending on how the connection was closed.  In all
-cases, it is possible to acknowledge packets that are received as normal, but
-other reactions might be preferable depending on how the connection was closed.
-An endpoint that is in a draining period MUST NOT send packets containing frames
-other than ACK, PADDING, or CONNECTION_CLOSE.
+in the draining period depending on how the connection was closed.  An endpoint
+that is in a draining period MUST NOT send packets unless they contain a
+CONNECTION_CLOSE or APPLICATION_CLOSE frame.
```

Comment:
This looks fine, but what's APPLICATION_CLOSE.  Did I miss that from another PR?

Reply:
See #722.

Reply:
I saw that PR go by, but the description didn't mention a new frame type.  I'm not super opposed, but I definitely wasn't expecting it.